impeller/entity/entity.cc

```diff
@@ -99,4 +102,64 @@ bool Entity::Render(const ContentContext& renderer,
   return contents_->Render(renderer, *this, parent_pass);
 }
 
+Vector4 min(Vector4 value, float threshold) {
+  return Vector4(std::min(value.x, threshold), std::min(value.y, threshold),
+                 std::min(value.z, threshold), std::min(value.w, threshold));
+}
+
+Color blendColor(const Color& src,
```

Comment:
I think this function is simple enough that a unit test makes sense

Reply:
Agreed, all of these CPU blends should be covered in an automated test.

@JsouLiang The only reason we're so light on tests in Impeller right now is that we don't want to flood Impeller with pixel tests that are impossible to maintain. Anything that can be easily tested on the CPU should get covered with automated tests. Anything that can't should at least be visible in a manual playground.

Reply:
Done